source/Element/Component.h

```diff
@@ -5,9 +5,9 @@ class Component
 {
 public:
 	Component(Entity* anOwner) : myOwner(anOwner) {};
```

Comment:
Can't Entity* be made const here?

Reply:
No, we might want to change the owners later inside of the components, so I think its better to leave it non const.